docs/create-packages/Select-assemblies-referenced-by-projects.md

```diff
@@ -9,25 +9,44 @@ ms.topic: conceptual
+Assemblies are used in two different ways during a build. The first is for compile, which allows the package consumer's code to compile against APIs in the assembly, and for Intellisense to give suggestions. The second is runtime, where the assembly is copied to the `bin` directory and is used at runtime.
 
-> [!Note]
-> Explicit assembly references are related to .NET assemblies. It is not a method to distribute native assemblies that are P/Invoked by a managed assembly.
+## Recommendation: Package dependencies and include/exclude assets
 
-## `PackageReference` support
+Our recommendation is to have one package per assembly, in which case you can add `compile` to the private assets. Note that the default private assets when none is supplied is `build;analyzers`. Therefore, you should use `PrivateAssets="compile;build;analyzers"` in your `PackageReference` or `ProjectReference`. This provides more flexibility and minimizes risk that two different packages will contain different versions of the same assembly, preventing NuGet from being able to manage versions.
 
-When a project uses a package with `PackageReference` and the package contains a `ref\<tfm>\` directory, NuGet will classify those assembles as compile-time assets, while the `lib\<tfm>\` assemblies are classified as runtime assets. Assemblies in `ref\<tfm>\` are not used at runtime. This means it is necessary for any assembly in `ref\<tfm>\` to have a matching assembly in either `lib\<tfm>\` or a relevant `runtime\` directory, otherwise runtime errors will likely occur. Since assemblies in `ref\<tfm>\` are not used at runtime, they may be [metadata-only assemblies](https://github.com/dotnet/roslyn/blob/main/docs/features/refout.md) to reduce package size.
+```xml
+<ItemGroup>
+  <ProjectReference Include="..\OtherProject\OtherProject.csproj" PrivateAssets="compile;build;analyzers" />
+  <PackageReference Include="SomePackage" Version="1.2.3" PrivateAssets="compile;build;analyzers" />
+</ItemGroup>
+```
 
-> [!Important]
-> If a package contains the nuspec `<references>` element (used by `packages.config`, see below) and does not contain assemblies in `ref\<tfm>\`, NuGet will advertise the assemblies listed in the nuspec `<references>` element as both the compile and runtime assets. This means there will be runtime exceptions when the referenced assemblies need to load any other assembly in the `lib\<tfm>\` directory.
+This gives your package consumers the flexibility to directly reference the package to get the compile time assets.
 
-> [!Note]
-> If the package contains a `runtime\` directory, NuGet may not use the assets in the `lib\` directory.
+## Not recommended: Multiple assemblies in one package
```

Comment:
If the not recommended is multiple assemblies in one package, shouldn't the top one be, recommended; one assembly per package?